karapace/kafka_rest_apis/__main__.py

```diff
@@ -1,29 +1,36 @@
+from contextlib import closing
 from karapace import version as karapace_version
+from karapace.config import read_config
 from karapace.kafka_rest_apis import KafkaRest
 
 import argparse
 import os
 import sys
 
 
-def main():
+def main() -> int:
```

Comment:
All the scripts are doing `sys.exit(main())` but most of the functions were missing a return with the error code.